packages/next/src/build/babel/loader/get-config.ts

```diff
@@ -273,6 +273,10 @@ function getFreshConfig(
       return false
     }
 
+    if (filename.includes('node_modules')) {
```

Comment:
This needs to include `/node_modules/` (but also `\node_modules\` for windows) there's some cases that check node_modules in the codebase already